src/preprocessing/ahn_preprocessing.py

```diff
@@ -262,8 +218,11 @@ def process_ahn_las_tile(ahn_las_file, out_folder='', resolution=0.1):
     grid_y, grid_x = np.mgrid[y_max-resolution/2:y_min:-resolution,
                               x_min+resolution/2:x_max:resolution]
 
-    ground_surface = _get_ground_surface(ahn_las, grid_x, grid_y)
-    building_surface = _get_building_surface(ahn_las, grid_x, grid_y)
+    # TODO Voor grond wil je smooth service (IDW). Voor gebouwen willen we upper threshold voor hoekjes voor gebouwen.
```

Comment:
TODO add text wat het verschil is tussen idw en.

En moet de max_dist anders zijn?

Reply:
Verschil is ook: voor grond wil je een soort gemiddelde (daarna +/- epsilon), terwijl je voor gebouwen liever een upper threshold wilt (en daarna alles onder die threshold als gebouw zien).

Reply:
En ja, die max_dist moet anders zijn :)

Reply:
Mooi opgelost zo trouwens!